### ISSUE TITLE
How to implement Subscription using Server InfFrame integration?

### ISSUE THREAD

**Issue by grim-reapper** [thephpleague/omnipay-sagepay#192]
How to implement subscription? I mean recurring payment where payment automatically detected after specific interval.

**Comment by judgej** [comment]
I don't believe SagePay/Opayo/NextNewName supports subsctiptions as a service. What it does support is [continuos authority](https://www.opayo.co.uk/support/account/merchant-numbers/general-setup/continuous-authority) and you can build your own subscription service around that.

To set up a continuous authority, you would take the user through a normal payment or authorisation, but with the `createToken` flag set. It's not super-intuitive, but if you check out the Opayo documentation, it should be a little clearer on how it maps onto this package.
